samples/wifi/softap/prj.conf

```diff
@@ -0,0 +1,86 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_WIFI=y
+CONFIG_WIFI_NRF700X=y
+
+# WPA supplicant
+CONFIG_WPA_SUPP=y
+
+# Softap
+CONFIG_NRF700X_AP_MODE=y
+CONFIG_WPA_SUPP_AP=y
+
+CONFIG_WPA_SUPP_LOG_LEVEL_INF=y
+
+# System settings
+CONFIG_NEWLIB_LIBC=y
+CONFIG_NEWLIB_LIBC_NANO=n
+
+# Networking
+CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_LOG=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_UDP=y
+CONFIG_NET_TCP=y
+
+CONFIG_NET_PKT_RX_COUNT=8
+CONFIG_NET_PKT_TX_COUNT=8
+
+# Below section is the primary contributor to SRAM and is currently
+# tuned for performance, but this will be revisited in the future.
+CONFIG_NET_BUF_RX_COUNT=16
+CONFIG_NET_BUF_TX_COUNT=16
+CONFIG_NET_BUF_DATA_SIZE=1500
+CONFIG_HEAP_MEM_POOL_SIZE=153600
```

Comment:
This is still old value it should either be 80K for minimal traffic or appropriately set for Zperf.